src/find-options/operator/Raw.ts

```diff
@@ -1,9 +1,47 @@
 import {FindOperator} from "../FindOperator";
+import {ObjectLiteral} from "../../common/ObjectLiteral";
 
 /**
  * Find Options Operator.
  * Example: { someField: Raw([...]) }
  */
-export function Raw<T>(value: string|((columnAlias: string) => string)): FindOperator<any> {
-    return new FindOperator("raw", value as any, false);
+export function Raw<T>(value: string): FindOperator<any>;
+
+/**
+ * Find Options Operator.
+ * Example: { someField: Raw((columnAlias) => `${columnAlias} = 5`) }
+ */
+export function Raw<T>(sqlGenerator: ((columnAlias: string) => string)): FindOperator<any>;
+
+/**
+ * Find Options Operator.
+ * For escaping parameters use next syntax:
+ * Example: { someField: Raw((columnAlias, parameters) => `${columnAlias} = ${parameters[0]}`, [5]) }
```

Comment:
I can't understand why do we need this syntax. Isn't it redundant?

Reply:
> I can't understand why do we need this syntax. Isn't it redundant?

Do you suggest leaving only parameters via ObjectLiteral and remove via array? I can do it.

Reply:
Yes, it's just seems redundant. It's better to left only one syntax if they both do same and there are no technical issues.

Reply:
> Yes, it's just seems redundant. It's better to left only one syntax if they both do same and there are no technical issues.

Done. Waiting for the tests to complete.